### DISCUSSION TITLE
Dag states overridden

### DISCUSSION THREAD

**Discussion by sanjeevjohal** [apache/airflow#31410] [Q&A]
I'm currently using v2.1.0 and managing our own Airflow instance ie not via AWS MWAA. Is it possible to indicate on the UI whenever a state gets overridden and by whom. Our use case is:

**TL;DR** We've deliberately decided now to reset any non-success states to success to keep our board clean and easier to see at a glance if there's been any failures.

**Comment by pankajastro** [comment]
if you just want to keep the dagrun state always success/green then you can add an end task EmptyOperator in your DAG with a trigger rule `all_done`
```
end = EmptyOperator(task_id="end",  trigger_rule =all_done)

[current_task] ->  end
```
https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/empty/index.html
https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#trigger-rules

**Reply by sanjeevjohal** [reply]
Thanks but you’re misunderstood my requirement. When I force a state change I want to know that it was overridden in the UI.

**Reply by pankajastro** [reply]
ok, what do you mean by "it was overridden in the UI"?
